packages/synapse-react-client/src/components/SynapsePortalBanners/SynapsePortalBanners.test.tsx

```diff
@@ -18,9 +25,26 @@ const defaultProps: SynapsePortalBannersProps = {
   sourceAppConfigTableID: 'syn7000',
 }
 
-vi.mock('../../synapse-client/SynapseClient', () => ({
-  getQueryTableAsyncJobResults: vi.fn(),
-}))
+vi.mock('@/synapse-queries', async () => {
+  const actual = await vi.importActual<typeof import('@/synapse-queries')>(
+    '@/synapse-queries',
+  )
+  return {
+    ...actual,
+    useGetEntityPath: vi.fn(),
+    useGetQueryResultBundleWithAsyncStatus: vi.fn(),
+  }
+})
```

Comment:
Mock could be brittle, it will not work if the import in the component is more specific than the import in this mock.